src/test/java/com/github/thorbenkuck/netcom2/network/server/RemoteObjectRegistrationImplTest.java

```diff
@@ -1,42 +1,355 @@
 package com.github.thorbenkuck.netcom2.network.server;
 
+import com.github.thorbenkuck.netcom2.annotations.rmi.RegistrationOverrideProhibited;
+import com.github.thorbenkuck.netcom2.exceptions.RemoteObjectInvalidMethodException;
+import com.github.thorbenkuck.netcom2.network.shared.comm.model.RemoteAccessCommunicationRequest;
+import com.github.thorbenkuck.netcom2.network.shared.comm.model.RemoteAccessCommunicationResponse;
 import org.junit.Test;
 
+import java.math.BigInteger;
+import java.util.UUID;
+
+import static com.github.thorbenkuck.netcom2.TestUtils.UUID_SEED_1;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
 public class RemoteObjectRegistrationImplTest {
+	@Test(expected=IllegalArgumentException.class)
+	public void registerInvalidArrayLength() throws Exception {
+		//Arrange
+		RemoteObjectRegistrationImpl remoteObjectRegistration = new RemoteObjectRegistrationImpl();
+		TestRemoteObject remoteObject = new TestRemoteObject();
+		Class[] classes = new Class[0];
+
+		//Act
+		remoteObjectRegistration.register(remoteObject, classes);
+
+		//Assert
```

Comment:
If the Exception is the required criteria for this test to succeed, a fail() at assert should be inserted.